### DISCUSSION TITLE
lagging

### DISCUSSION THREAD

**Discussion by christorpher** [Budibase/budibase#9153] [Help]
![Capture](https://user-images.githubusercontent.com/120565326/209160945-a9cc1c2d-fbbe-49a3-902d-0a5b55a57eeb.PNG)

**Comment by christorpher** [comment]
![Capture1](https://user-images.githubusercontent.com/120565326/209161200-4df728c7-9b38-4003-8f4b-a4b16d364879.PNG)

**Comment by christorpher** [comment]
my bb is lagging unable to see the form instantly

**Comment by christorpher** [comment]
it takes minimum 15 mins to show  a complete form 
(there are no row entry is newly desgined and published form)

**Reply by mike12345567** [reply]
Hi @christorpher is this in the Budibase Cloud or self-hosted? Also could you provide an app export exhibiting these behaviours?

**Comment by christorpher** [comment]
its self hosted

**Comment by christorpher** [comment]
[NG01-export-1671691375895 (1).tar.gz](https://github.com/Budibase/budibase/files/10287999/NG01-export-1671691375895.1.tar.gz)

**Reply by mike12345567** [reply]
Hi @christorpher - I tried importing this app into the most recent version of Budibase and I could not find any issues with it. Two things from here:
1. It would be worth confirming you are utilising the latest version of Budibase in your self host installation
2. Can you provide more in-depth reproduction instructions based on the app export? I believe you are having issues with the `Checklist` screen - however after publishing the app and going through all of the screens I did not experience any slow down at all - with load times being consistently below a second.

**Comment by christorpher** [comment]
The version on budibase is 2.2.3

**Reply by mike12345567** [reply]
Hi @christorpher - there has been a few releases since this - with the latest being 2.2.9 - it may be worth trying these patch fixes.

**Comment by christorpher** [comment]
Sir can you please guide me the producer as I tried the button on budibase
it says updated I m running it through dockers

**Comment by christorpher** [comment]
Procedure *^
